drag_and_drop_v2/public/css/drag_and_drop.css

```diff
@@ -100,20 +100,20 @@
     align-items: center;
 
     position: relative;
-    border: 1px solid rgba(0, 0, 0, 0.1);
     border-radius: 3px;
     padding: 0; /* padding: 5px looks better but makes some blocks to change in size when dropped onto the target; */
 }
 
 .xblock--drag-and-drop .drag-container .option {
     display: inline-block;
-    width: auto;
+    width: 150px;
```

Comment:
Should we be hardcoding the size of these here?
The width can be set as a percentage in the studio editor, and this would likely override that.

Reply:
@kaizoku Thanks for the review. I have pushed the said changes except the width of tiles as they are fixed in the deisgn. You probably don't have all the changes for drag and drop in this PR so the tiles are not fitting in. Please let his PR merged in drag-and-drop-v4 first to have the final results.

Thanks